lib/internal/source_map/source_map_cache.js

```diff
@@ -254,6 +255,9 @@ function sourceMapFromDataUrl(sourceURL, url) {
 function sourcesToAbsolute(baseURL, data) {
   data.sources = data.sources.map((source) => {
     source = (data.sourceRoot || '') + source;
+    if (isAbsolute(source)) {
```

Comment:
Would you mind updating the comment of the function `sourcesToAbsolute` to reflect the change?

Reply:
I thought about this, but it seems to me(not a native English speaker) that the existing comments are already well-suited for the fixed logic rather than for the logic before the fix. 😂

> // If the sources are not absolute URLs after prepending of the "sourceRoot",
// the sources are resolved relative to the SourceMap (like resolving script
// src in a html document).

Reply:
What do you think about the following change?

```diff
  // If the sources are not absolute URLs after prepending of the "sourceRoot",
  // the sources are resolved relative to the SourceMap (like resolving script
  // src in a html document).
+ // If the sources are absolute paths, the sources are converted to absolute file URLs.
```

Reply:
@legendecas Great! fixed.